ios/VizbeeNativeManager.m

```diff
@@ -358,6 +384,9 @@ -(void) onSessionStateChanged:(VZBSessionState)newState {
     if (newState == VZBSessionStateConnected) {
         [self addVideoStatusListener];
         [self addVolumeStatusListener];
+
+        NSMutableDictionary* dummySignInInfoMap = [NSMutableDictionary new];
+        [self sendEvent:@"VZB_INVOKE_GET_SIGNIN_INFO" withBody:dummySignInInfoMap];
```

Comment:
Same as above, set's use contacts instead of magic numbers or string

Reply:
All magic strings are moved to VizbeeConstants file